portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase

```diff
@@ -1472,6 +1472,104 @@ definition {
 		}
 	}
 
+	@description = "LPS-128600 TC-3. User can't log in with invalid custom field mapping on SP. Blocked by LPS-142222."
+	@priority = "4"
+	test CannotLoginWithInvalidCustomFieldMappingInSAML {
+		property app.server.bundles.size = "0";
+		property portal.upstream = "quarantine";
```

Comment:
This macro requires the idpURL from default , I just tweaked it to check if we are already logged in or not to be able to skip the login process.
Are you suggesting to create another if else part with having a default url pointing to localhost? Because then we should extend the other macros here too, and I suggest we should do that in a different ticket